crates/solidity/outputs/npm/package/wasm/generated/interfaces/nomic-foundation-slang-parser.d.ts

```diff
@@ -72,6 +72,10 @@ export class Parser {
    * Creates a new parser instance for the specified language version.
```

Comment:
Now that #1257 is merged, I suggest we add a new H2 section to `# 4.2. Choosing a Solidity Version` to talk about this API, and to give an example on how to use it. I'm happy to help with this as well if you like.